### PR TITLE
Add EMIT_INTERVAL parameter

### PR DESCRIPTION
**Pull request by gtricot**
Hi @fmartinou,

First of all, thank you for that component. I'm new to HomeAssistant and it works like a charm.
I forked your repo and created that PR because I'd like to configure the interval between 2 MQTT emissions (Update of my Teleinfo sensors every 5 or 10 seconds seems sufficient in my own opinion).
Unfortunately, I don't know how to build a Docker image locally and test it before pushing my new code.
Could you please help me to achieve that ?

Thank's.

### CODE REVIEW

**Comment by fmartinou** [comment]
Hi @gtricot ,

Thank you very much for your contribution!

To locally build  the Docker image, you need to:
1. [Install Docker](https://docs.docker.com/get-docker/) on your machine
2. Build the image `docker build -t teleinfo-mqtt-local .` (run the command from the root folder of the project)
3. Check the image `docker run ... teleinfo-mqtt-local`

Please notice that you don't need to build the image to be able to test the app.
(you can just run the node.js app with `npm start`)

If you can just test your changes by running the node app itself, it will be ok.

I'll review your changes and will add comments if so.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Configuration uses environment variables.
 |MQTT_PASSWORD    | MQTT password (optional)                  |                        |
 |IDENTIFIER       | Identifier for Home-Assistant Discovery   |                        |
 |DISCOVERY_PREFIX | Topic prefix for Home-Assistant Discovery | homeassistant          |
+|EMIT_INTERVAL    | Interval between 2 MQTT emissions (ms)    | 0                      |
```
Comment (fmartinou) [inline README.md:36]: Could you add a little explanation to specify that `0` means that all frames will be published (no sampling)?

**Comment by fmartinou** [comment]
FYI, the CI is managed by Travis: https://travis-ci.org/github/fmartinou/teleinfo-mqtt/builds/763571285
One unit test is failing; may you adjust it?
Thanks!

**Comment by fmartinou** [comment]
Please forget my comment about failed unit tests; I didn't look at the logs in details.
Actually, the UT are ok :)
The build fails for another reason (an issue with docker credentials); there is no relation with your changes.

**Comment by gtricot** [comment]
After a little changed (EMIT_INTERVAL in seconds), I tested my changes running the node app locally and it works as expected : 
- EMIT_INTERVAL=0 : All frames are emitted to MQTT topic (if different)
- EMIT_INTERVAL=10 : Frames are emitted to MQTT topic every 10 seconds (only if different too)

Could you please review my changes ?

**Review by fmartinou — APPROVED** [review]
LGTM

**Comment by fmartinou** [comment]
I just published the release [`3.1.0`](https://hub.docker.com/layers/fmartinou/teleinfo-mqtt/3.1.0/images/sha256-6cbb0d5b5e7d4a9e4e0a5b3067644861e885740e61b74a2a2ad2699bf00b0691?context=explore) wich includes your changes.
Thank you for your contribution!

**Pull request merged**